2020/day9/solution.py

```diff
@@ -0,0 +1,33 @@
+
+def find_first_exception(numbers: list[int], lookback_count : int) -> int:
+    window = set(numbers[:lookback_count])
+
+    def window_sums_to_value(window, value):
```

Comment:
```suggestion
    def window_has_summands_to_value(window, value):
```
The name made me think the implementation should be `sum(window) == value`.

Another option is to inline this given it's a one-liner used in one place (and maybe add a comment) which I would probably personally opt for.